src/bases.jl

```diff
@@ -205,7 +234,7 @@ end
 Compute the basis vectors of the tangent space at a point on manifold `M`
 represented by `p`.
 
-Returned object derives from [`AbstractBasis`](@ref) and may have a field `.vectors`
+Returned object derives from [`AbstractBasis`](@ref) and may have a 𝔽 `.vectors`
```

Comment:
```suggestion
Returned object derives from [`AbstractBasis`](@ref) and may have a field `.vectors`
```
This refers to a field in a struct.